frontend/packages/patternfly/src/components/notification-drawer/notification-drawer.tsx

```diff
@@ -3,6 +3,7 @@ import {
   Drawer,
   DrawerContent,
   DrawerPanelContent,
+  DrawerPanelBody,
 } from '@patternfly/react-core/dist/js/experimental/components/Drawer';
```

Comment:
TODO: I thought patternfly switched from 'experimental' to 'beta'.  We probably need to change this import before merge!